### PR TITLE
Document new rest api - getLastMessageId

### PR DESCRIPTION
**Pull request by legendtkl**
Issue ticket: https://github.com/apache/pulsar/issues/3233

### CODE REVIEW

**Comment by legendtkl** [comment]
hi @sijie , would you help review this pr, thanks

**Comment by legendtkl** [comment]
rerun java8 tests

File: site/docs/latest/admin-api/persistent-topics.md [hunk 1]
```diff
@@ -644,3 +644,10 @@ String topic = "persistent://my-tenant/my-namespace/my-topic";
 String subscriptionName = "my-subscription";
 admin.persistentTopics().deleteSubscription(topic, subscriptionName);
 ```
+
+### Last Message Id
+
+It gives the last commited message ID for a persistent topic.
+
+#### REST API
```
Comment (sijie) [inline site/docs/latest/admin-api/persistent-topics.md]: typically for a given rest endpoint, it should be documented for 3 parts, 1) http rest api 2) pulsar-admin 3) admin java api. Can you follow what other endpoints do?

Also did we expose the endpoint to admin java api?

**Comment by legendtkl** [comment]
I update the document. Would you help review it again? @sijie

**Comment by sijie** [comment]
well done @legendtkl

**Pull request merged**